<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Practice 3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabBarMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>430</width>
      <height>430</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>430</width>
      <height>430</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>430</width>
      <height>430</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>430</width>
      <height>430</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabOverview">
     <attribute name="title">
      <string>Overview</string>
     </attribute>
     <widget class="QTableView" name="tblMainOverview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>405</width>
        <height>350</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>405</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>405</width>
        <height>350</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>405</width>
        <height>350</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
     </widget>
     <widget class="QLabel" name="lblExportLeadingCountry">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>370</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Leading exporter:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outExportLeadingCountry">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>370</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="maxLength">
       <number>40</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabImportProductReport">
     <attribute name="title">
      <string>Report of imports of product</string>
     </attribute>
     <widget class="QLineEdit" name="iptImportProductReport">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>40</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tblImportProductReport">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>405</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>405</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>405</width>
        <height>300</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>405</width>
        <height>300</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
     </widget>
     <widget class="QLabel" name="lblImportProductReport">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Enter the product for the impot report:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblImportProductReportSummary">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Total volume of transactions:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outImportProductReportSummary">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>370</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="maxLength">
       <number>40</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabImportCountryReport">
     <attribute name="title">
      <string>Country report</string>
     </attribute>
     <widget class="QTableView" name="tblImportCountryReport">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>405</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>405</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>405</width>
        <height>300</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>405</width>
        <height>340</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
     </widget>
     <widget class="QLabel" name="lblImportCountryReport">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Enter the country for the impot report:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iptImportCountryReport">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>40</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblImportCountryReportSummary">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Total volume of transactions:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outImportCountryReportSummary">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>370</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="maxLength">
       <number>40</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabGraphExport">
     <attribute name="title">
      <string>Graph export</string>
     </attribute>
     <widget class="QLineEdit" name="iptGraphExport">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="maxLength">
       <number>40</number>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblGraphExport">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Enter the product for which graph for:</string>
      </property>
     </widget>
     <widget class="QWidget" name="wdgGraphExport" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>401</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>450</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_iptImportProductReport_enter()</slot>
  <slot>on_iptImportCountryReport_enter()</slot>
  <slot>on_iptGraphExport_enter()</slot>
 </slots>
</ui>
